extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -361,6 +383,7 @@ export class AmpStoryStoreService {
       [StateProperty.STORY_HAS_AUDIO_STATE]: false,
       [StateProperty.SYSTEM_UI_IS_VISIBLE_STATE]: true,
       [StateProperty.UI_STATE]: UIType.MOBILE,
+      [StateProperty.ACTIONS_WHITELIST]: [],
```

Comment:
Consider adding a comment explaining why this needs to be initialized to empty array.

Reply:
Done

Reply:
I was thinking something like "amp-story only allows actions on a case-by-case basis to preserve UX behaviors. By default, no actions are allowed." No big deal.